cypress/integration/score_card_spec.js

```diff
@@ -18,6 +18,53 @@ describe('Scorecard user flow', () => {
       .get('p').contains('2')
       .get('.plus-btn').eq(1).click()
       .get('p').contains('4')
+      .get('.next-hole-btn').click()
```

Comment:
Not very familiar with cypress testing, just curious if there is a better way to refactor this ? Maybe a thought for later on.